tests/test_integrated_channels/test_cornerstone/test_views.py

```diff
@@ -47,14 +51,6 @@ def test_course_list_unauthorized_non_customer(self):
         response = self.client.get(self.course_list_url)
         self.assertEqual(response.status_code, 401)
 
-    def test_course_update_unauthorized_non_customer(self):
```

Comment:
why was this removed?

Reply:
this is for the update version of the endpoint which we're removing in this PR as well. there is a version of this test for the list endpoint - the one in use - immediately above this.